testsuite/gnat2goto/tests/character_literal/character_literal.adb

```diff
@@ -0,0 +1,6 @@
+procedure Character_Literal is
+   A : Character := 'a';
+   B : Character := 'b';
+begin
+   pragma Assert (A > B);
```

Comment:
If not essential, I'd suggest removing the pragma. It's not a requirement of the failure case.

Reply:
Will remove, it doesn't make sense here.

Reply:
FWIW I think it's worth keeping it in here as it'll require less changes when converting this to a real case

Reply:
@hannes-steffenhagen-diffblue  
If so: "If you keep it - I'd suggest a pragma that would pass".   No?